requirements.txt

```
@@ -2,5 +2,6 @@ ops ==2.16.0
 requests ==2.32.3
 jsonschema==4.23.0
 cryptography >= 3.4.8
-lightkube===0.15.0
+lightkube>=0.15.0
```

Cosl (see below) requires version newer than 0.15.0.

---

If it requires newer than 0.15.0 than this should be 
```suggestion
lightkube>0.15.0
```